File: ApiSuvesaPos/DatosFE/Class/Configuraciones.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using DatosFE.Models;

namespace DatosFE.Class
{
    class Configuraciones
    {
        private FEContext entities;

        public Configuraciones()
        {
            entities = new FEContext();
        }

        public int CrearConfiguracion(Configuracione Conf) // crear Configuraciones apartir de objeto de creacion
        {
            try
            {
                entities.Configuraciones.Add(Conf);
                return entities.SaveChanges();

            }
            catch (Exception ex)
            {

                throw ex;
            }

        }

        public int BorrarConfiguracion(int id) // borrar Configuraciones apartir del id Actividad


        {
            try
            {
                var p = entities.Configuraciones.Find(id);
                entities.Remove(p);
                return entities.SaveChanges();

            }
            catch (Exception ex)
            {

                throw ex;
            }

        }

        public List<Configuracione> ObtenerActividades() //obtener lista actividades 
        {
            try
            {
                var temp = from c in entities.Configuraciones

                           select c;
                List<Configuracione> result = temp.ToList<Configuracione>();

                if (result.Count > 0)
                {
                    return result;
                }
                else
                {
                    return result = null;
                }


            }
            catch (Exception ex)
            {
                throw ex;
            }
        }
    }
}

File: ApiSuvesaPos/NegocioFE/Logica/Clase/Configuraciones.vb
Namespace Logica
    Public Class Configuraciones
        Private db As New DatosFE.Class.Configuraciones


        Public Function Crear(conf As DatosFE.Models.Configuracione) As String
            Return Me.db.CrearConfiguracion(conf)
        End Function

        Public Function Borrar(id As Integer) As String
            Return Me.db.BorrarConfiguracion(id)
        End Function

        Public Function Obtener(cedula As String) As List(Of DatosFE.Models.Configuracione)
            Return Me.db.ObtenerConfiguraciones(cedula)
        End Function

    End Class
End Namespace